src/docs/guides/laravel.md

```diff
@@ -0,0 +1,185 @@
+1. Fork the basic <a href="https://github.com/railwayapp-templates/laravel" target="_blank">Laravel GitHub repo</a>. 
+    - If you already have a GitHub repo you want to deploy, you can skip this step.
+2. Create a <a href="https://railway.app/new" target="_blank">New Project.</a>
+3. Click **Deploy from GitHub repo**.
+4. Select the `laravel` or your own GitHub repo.
+    - Railway requires a valid GitHub account to be linked. If your Railway account isn't associated with one, you will be prompted to link it.
+5. Click **Add Variables**. 
+    - Add all your app environment variables.
+5. Click **Deploy**.
+
+Once the deployment is successful, a Railway [service](/guides/services) will be created for you. By default, this service will not be publicly accessible.
+
+To set up a publicly accessible URL for the service, navigate to the **Networking** section in the [Settings](/overview/the-basics#service-settings) tab of your new service and click on [Generate Domain](/guides/public-networking#railway-provided-domain).
+
+<Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1727885952/docs/quick-start/CleanShot_2024-10-02_at_17.18.04_2x_nn78ga.png"
+alt="screenshot of the deployed Laravel service showing the Laravel home page"
+layout="responsive"
+width={2855} height={2109} quality={100} />
+
+## Deploy from the CLI
+
+1. <a href="/guides/cli#installing-the-cli" target="_blank">Install</a> and <a href="/guides/cli#authenticating-with-the-cli" target="_blank">authenticate with the CLI.</a>
+2. Clone the forked <a href="https://github.com/railwayapp-templates/laravel" target="_blank">Laravel GitHub repo</a> and `cd` into the directory. 
+    - You can skip this step if you already have an app directory or repo on your machine that you want to deploy.
+3. Run `railway init` within the app directory to create a new project. 
+4. Run `railway up` to deploy.
+    - The CLI will now scan, compress and upload our Laravel app files to Railway's backend for deployment.
+
+## Deploy via Custom Scripts
+
+This setup deploys your Laravel app on Railway, ensuring that your database, scheduled tasks (crons), and queue workers are all fully operational.
+
+The deployment structure follows a "majestic monolith" architecture, where the entire Laravel app is managed as a single codebase but split into four separate services on Railway:
+- **App Service**: Handles HTTP requests and user interactions.
+- **Cron Service**: Manages scheduled tasks (e.g., sending emails or running reports).
+- **Worker Service**: Processes background jobs from the queue.
+- **Database Service**: Stores and retrieves your application's data.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1727910244/docs/quick-start/deploy%20architecture.png"
+alt="screenshot of the deploy architecture of the Laravel app"
+layout="responsive"
+width={3118} height={1776} quality={100} />
+_My Majestic Monolith Laravel app_
+
+Please follow these steps:
```

Comment:
I think the phrasing here could be improved a bit. As a user reading this I'm a bit unsure _why_ I need to make these scripts. I think the guide could do a better job explaining what is actually happening. Right now it feels a bit like "trust me just do this".